packages/graphql-key-transformer/src/KeyTransformer.ts

```diff
@@ -160,7 +161,7 @@ export default class FunctionTransformer extends Transformer {
         if (args.fields.length > 2) {
             const compositeKeyFieldNames = args.fields.slice(1);
             const compositeKeyFields = definition.fields.filter(field => Boolean(compositeKeyFieldNames.find(k => k === field.name.value)));
-            const keyName = args.name || 'Primary';
+            const keyName = Case.pascal(args.name || 'Primary');
```

Comment:
Is this a breaking change?

Reply:
This keyName variable is only used for the GraphQL input and condition types, nothing that's directly referenced like DynamoDB GSI name, so I don't think it is a breaking change.